modules/5_install/templates/install_vars.yaml

```diff
@@ -52,3 +52,29 @@ bastion_vip: "${bastion_vip}"
 
 cni_network_provider: ${cni_network_provider}
 cni_network_mtu: ${cni_network_mtu}
+
+%{ if luks_compliant && length(luks_config) > 0 ~}
+luks:
+ enabled: true
+ config:
+%{ for item in luks_config ~}
+    - thumbprint: ${item.thumbprint}
+      url: ${item.url}
+%{ endfor ~}
+ filesystem_device: ${luks_filesystem_device}
+ format: ${luks_format}
+ wipeFileSystem: "${luks_wipeFileSystem}"
+ device: ${luks_device}
+ label: ${luks_label}
+%{ if luks_options != "" }
+ options:
+%{ for item in luks_options ~}
+    - ${item}
+%{ endfor ~}
+%{ endif ~}
+ wipeVolume: "${luks_wipeVolume}"
+ name: ${luks_name}
+%{ else ~}
+luks:
+ enabled: false
+%{ endif ~}
```

Comment:
Added required changes as suggested in below PR:
https://github.com/ocp-power-automation/ocp4-upi-powervs/pull/441